modules/reference/pages/configuration-parameters.adoc

```diff
@@ -1265,8 +1265,7 @@ primary cluster’s IPs, separator by `,' |`nan`
 |System.CrossRegionReplication.PrimaryKafkaPort |Kafka mirrormaker
 primary cluster’s KafkaPort |`30002`
 
-|System.CrossRegionReplication.TopicPrefix |The prefix of GPE/GUI/GSQL
-Kafka Topic, by default is empty. |`nan`
+|System.CrossRegionReplication.TopicPrefix |The prefix of GPE/GUI/GSQL Kafka Topic, by default is empty.For details of what it does and how it is used, please refer to xref:tigergraph-server:cluster-and-ha-management:set-up-crr.adoc[].|`nan`
```

Comment:
```suggestion
|System.CrossRegionReplication.TopicPrefix |The prefix of GPE/GUI/GSQL Kafka Topic, by default is empty.
For details of what it does and how it is used, please refer to xref:cluster-and-ha-management:set-up-crr.adoc[].|`nan`
```

Reply:
@anjaliIthapeTG 
Added missing space between sentences.
Removed component name (tigergraph-server) from xref, because reference is within the same component.